### DISCUSSION TITLE
Not receiving payload in pong listener

### DISCUSSION THREAD

**Discussion by sebovzeoueb** [uNetworking/uWebSockets.js#827] [Q&A]
I'm trying to set up a simple system with pinging the client to measure latency, and I'm using ws.ping(buffer) to do so. The pong listener is triggering correctly, so the browser is ponging, but the payload is undefined. Any idea why this might be?

**Comment by mikrosystheme** [comment]
Reading the source code of the app wrapper (https://github.com/uNetworking/uWebSockets.js/blob/master/src/AppWrapper.h#L215-L224), it looks like the javascript callback for the pong event is applied to only one argument (the ws object). The message is received on the C++ side but not forwarded to JS.

**Reply by sebovzeoueb** [reply]
Oh OK, it sounds like that should be fixed!
